plugins/view-transitions/includes/settings.php

```diff
@@ -270,19 +279,30 @@ static function (): void {
 			'title'       => __( 'Post Content Selector', 'view-transitions' ),
 			'description' => __( 'Provide the CSS selector to detect the post content element.', 'view-transitions' ),
 		),
+		'enable_admin_transitions'     => array(
+			'title'       => __( 'Use transition in WP Admin', 'view-transitions' ),
```

Comment:
Maybe make the title shorter as just "WP Admin"? Since the description is used as the label for the checkbox, I think it provides enough context, and putting too much of it in the `title` too causes duplication.
```suggestion
			'title'       => __( 'WP Admin', 'view-transitions' ),
```